ipaserver/install/service.py

```diff
@@ -578,7 +623,13 @@ def run_step(message, method):
         self.steps = []
 
     def ldap_enable(self, name, fqdn, dm_password=None, ldap_suffix='',
```

Comment:
Devil's advocate: this is changing the IPA API. Yes, it's internal but we don't know if any 3rd parties rely on it. Should the calls to ldap_enable be changed to a new method ldap_<something else>?

Reply:
Good point, I kept the old ``ldap_enable`` call and changed all other places to use the new method ``ldap_configure``.